### PR TITLE
Enhanced Feedback Management: Introducing Issue Templates

### LINKED ISSUE #433: Feature of Issue-froms
### Description
Leaderboard is a tool that will be used by many organizations and they might be facing any issue and requesting new features,
so to take the feedback we need to create an `issue form` in the leaderboard repository.

**We need to add mainly two forms** 
1. Report a Technical/Visual Issue on the Leaderboard Website
2. Suggest a new feature or improvement for the Leaderboard Website

### Example
image

### Refrences: https://github.com/nodejs/nodejs.org/issues/new/choose

### PR DESCRIPTION
**Pull request by dgparmar14**
## Proposed Changes

Add support for issue-form in the leaderboard repository to track feedback and bug reports from other organizations. This involves creating a folder `.github/ISSUE_TEMPLATE` and adding two types of forms:

1. **Feature Request:** `feature-request.yml`
2. **Technical Visual:** `technical-visual.yml`

## Description

This PR addresses the need to streamline the feedback and bug reporting process by implementing issue forms within the repository. By introducing structured templates for different types of issues, such as feature requests and technical visuals issues, we aim to gather comprehensive information from users and stakeholders.

## Changes Made

- Added `.github/ISSUE_TEMPLATE` folder.
- Included two YAML templates: `feature-request.yml` and `technical-visual.yml`.

## Resolved Issue

Fixes #433

### CODE REVIEW

File: components/Footer.tsx
Comment (rithviknishad) [inline components/Footer.tsx:1]: Why are there changes present in the footer? Unrelatd to the issue

**Review by rithviknishad — CHANGES_REQUESTED** [review]
Issue forms lgtm... but...

**Comment by rithviknishad** [comment]
Also, it'd be great if you could give a more meaningful title for your PRs 😄

File: components/Footer.tsx
Comment (dgparmar14) [reply components/Footer.tsx:1]: You suggested me there will be a hardcoded link on the website. So I place it at the footer

Comment (rithviknishad) [reply components/Footer.tsx:1]: Oh yea missed that, then could you clear the conflicts and update the title to convey this too?

**Review by rithviknishad — CHANGES_REQUESTED** [review]
This does not look good. Could you rethink the UI a bit and come up with a better layout?

image

image

**Comment by rithviknishad** [comment]
https://tailwindui.com/components/marketing/sections/footers

**Review by rithviknishad — APPROVED** [review]

**Pull request merged**